CMakeLists.txt

```diff
@@ -642,4 +642,12 @@ configure_package_config_file(${CMAKE_CURRENT_SOURCE_DIR}/Config.cmake.in
   INSTALL_DESTINATION "share/s2/"
   NO_SET_AND_CHECK_MACRO
   NO_CHECK_REQUIRED_COMPONENTS_MACRO)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/s2Config.cmake DESTINATION "share/s2/")
+
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/s2ConfigVersion.cmake"
+  VERSION ${PROJECT_VERSION}
+  COMPATIBILITY SameMinorVersion)
```

Comment:
We want `SameMinorVersion` while we're at 0.x but then `SameMajorVersion` when we go to 1.x.  Is it going to cause any problems to change later?

Reply:
I guess it won't?

Another option may be to set compatibility to `AnyNewerVersion` and give users the freedom and responsibility to deal with breaking changes.